lib/parseCommandLineOptions.js

```diff
@@ -26,6 +26,23 @@ module.exports = function parseCommandLineOptions(argv) {
       type: 'string',
       demand: false,
     })
+    .options('browsers', {
+      describe:
+        "Specify which browsers to support. Controls which font formats to provide, and whether to apply a JavaScript-based polyfill for preloading fonts. Defaults to browserslist's default query: https://github.com/browserslist/browserslist#best-practices",
```

Comment:
This description doesn't help the user to discover that there is also support for all the browserslist configuration formats. How about:

```suggestion
        "Specify which browsers to support. Controls which font formats to provide, and whether to apply a JavaScript-based polyfill for preloading fonts. Defaults to browserslist's default query: https://github.com/browserslist/browserslist#best-practices",
        "Override your projects browserslist configuration to specify which browsers to support. Controls font formats and polyfill. Defaults to browserslist's default query if your project has no browserslist configuration"
```